packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconEllipses from 'terra-icon/lib/icon/IconEllipses';
+import 'terra-base/lib/baseStyles';
+import CollapsibleMenuViewItem from './CollapsibleMenuViewItem';
+import CollapsibleMenuViewItemGroup from './CollapsibleMenuViewItemGroup';
+import CollapsibleMenuViewToggle from './CollapsibleMenuViewToggle';
+import CollapsibleMenuViewDivider from './CollapsibleMenuViewDivider';
+import styles from './CollapsibleMenuView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Item/ItemGroup/Divider elements to display in the view. Items that will fit in the space available will
+   * display face-up the remainder will be rolled into a menu
+   */
+  children: PropTypes.node.isRequired,
+
+  /**
+   * Width for the menu
```

Comment:
Can we add these width to the description? Looking at the props table, it only says menu width is a string